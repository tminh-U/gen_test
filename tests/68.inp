6
13 20 16 12 1 1 
6
5 6
4 5
4 4
1 3
2 5
4 4
