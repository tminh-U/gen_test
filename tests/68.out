4
3
2
6
5
2
